### DISCUSSION TITLE
I can't get the Serial Number. Please help me

### DISCUSSION THREAD

**Discussion by coderJoon** [PicoQuant/snAPI#14] [Q&A] [answered]
I just ran your demo code, but the serial number I wanted didn't come out. What should I do?

And, I don't even see any log records. please help me,,,

![image](https://github.com/PicoQuant/snAPI/assets/87929279/8a42501b-5b91-4f5e-9cde-6866b9c26998)
![image](https://github.com/PicoQuant/snAPI/assets/87929279/28c95cae-6af9-4385-95f3-0a23c1c3b482)
![image](https://github.com/PicoQuant/snAPI/assets/87929279/05b79dcd-9724-4313-b7d8-b9694d1f8686)

**Comment by coderJoon** [comment, marked as answer]
Solved!!! It's Just my fault. I'm sorry,,,!!

**Comment by tpoint75** [comment]
Nice! Is there anything that we could add to the documentation to prevent this fault?
